scripts/base_classes.py

```diff
@@ -88,6 +88,21 @@ def base_output_dirs(self):
 
         return (output_base, docker_output_base)
 
+    def check_chrom_prefix(self, bed_file):
+        """Raises exception if chromosome prefix doesn't match chromsome in bed file"""
+        chromosome_names = [x for x in range(1, 23)] + ["X", "Y", "M", "mt"]
+        chromosomes = [f"{self.settings['chromosome_prefix']}{chromsome}\t" for chromsome in chromosome_names]

```

Comment:
mt?

Reply:
mitochondrial, I'm pretty sure I've seen it as "mt" rather than "chrM"